docling/backend/html_backend.py

```diff
@@ -536,9 +534,32 @@ def handle_figure(self, element: Tag, doc: DoclingDocument) -> None:
     def handle_image(self, element: Tag, doc: DoclingDocument) -> None:
         """Handles image tags (img)."""
         _log.debug(f"ignoring <img> tags at the moment: {element}")
-
         doc.add_picture(
             parent=self.parents[self.level],
             caption=None,
             content_layer=self.content_layer,
         )
+        
+    def handle_anchor(self, element: Tag, doc: DoclingDocument) -> None:
+        """Handles anchor tags (<a>) by extracting the visible text and href attribute."""
+        # Extract the anchor text and the URL (href attribute)
+        text = element.get_text().strip()
+        href = element.get("href", "").strip()
+        
+        if text:
+            # Combine the text with the hyperlink if available
+            combined_text = f"{text} (Link: {href})" if href else text
+            doc.add_text(
```

Comment:
Very nice, but can you please do the hyperlink (see here for inspiration: https://github.com/docling-project/docling-core/blob/1af07218e1496029693eb7dcebd6f4ac4451003d/test/test_docling_doc.py#L824)

Reply:
Done 😄